cards/src/main/java/com/handonn/finapp/cards/service/ICardService.java

```diff
@@ -0,0 +1,21 @@
+package com.handonn.finapp.cards.service;
+
+
+import com.handonn.finapp.cards.model.CardDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+public interface ICardService {
+
+    void create(CardDto cardDto);
+
+    List<CardDto> getAllCard();
+
+    CardDto getByCardNumber(String cardNumber);
+
+    boolean updateCard(Long id, CardDto cardDto);
```

Comment:
update should return void, because we already implemented the ExceptionHandler. When update fail, it should throw error